brevbaker-api-model-common/src/testFixtures/kotlin/no/nav/brev/brevbaker/FellesFactory.kt

```diff
@@ -0,0 +1,60 @@
+package no.nav.brev.brevbaker
+
+import no.nav.pensjon.brevbaker.api.model.Bruker
+import no.nav.pensjon.brevbaker.api.model.Felles
+import no.nav.pensjon.brevbaker.api.model.Foedselsnummer
+import no.nav.pensjon.brevbaker.api.model.NavEnhet
+import no.nav.pensjon.brevbaker.api.model.SignerendeSaksbehandlere
+import no.nav.pensjon.brevbaker.api.model.Telefonnummer
+import java.time.LocalDate
+
+object FellesFactory {
```

Comment:
Har vi ikke en factory for Felles fra før?

Reply:
Glem det, ser at den blir slettet.

Reply:
Jepp. Denne klassa er eigentleg ei omdøyping av den som heita Fixtures, det vart knot frå Fixtures-klassar i andre modular som ville kalle denne, så da vart det lettare å døype om denne. Og synes ikkje Fixtures var så dekkjande for kva denne gjorde heller.